### ISSUE TITLE
Testing 2.1.7a

### ISSUE THREAD

**Issue by solbu** [blackadderkate/weather-widget-2#29]
I am testing in a virtual machine, and it looks good.

You write in your 2.1.7a release notes about a search button.
I see 2 +OMW and +met.no buttons.
When i click the +met.no button I see the «old» lat, lon, alt, url and place identifier fields – but no search button anywhere.

Where exactly is that search supposed to show up?

**Comment by blackadderkate** [comment]
On my computer the search button is above the OK button:
![Screenshot_20220205_003823](https://user-images.githubusercontent.com/83865293/152621233-7891d0bd-5f69-45bc-b5bf-374b114bb01d.png)

I have found some bugs with some timezones (such as Thailand) so I've just been doing some further hacking, and also made some extra changes to the dialog box (see below - it's not finished yet though). I'm hoping to release another Alpha version on Sunday...
![Screenshot_20220205_004537](https://user-images.githubusercontent.com/83865293/152621602-04c21f1c-cd0e-40c1-846d-9b8b9c48ff8d.png)
.

**Comment by solbu** [comment]
I should note that I compile from source – _Not_ the plasmoind binary – , as I am intending on packaging this for Mageia.
The source link I use is the source code tarball that github automatically adds to any releases.

I just noticed that the reported plasmoid version in the gui is «2.0.6».
So I think you are doing something wrong in how you do releases.
You need to provide the actuall source code as one of the items in the release assets.

**Comment by solbu** [comment]
Using this command to generate a tarball from the search branch, I made it work.
«git archive --prefix=weather-widget-2-2.1.7a/ HEAD -o weather-widget-2-2.1.7a.tar.gz»

**Comment by blackadderkate** [comment]
That release was something I uploaded especially for you, to show you what I was working on - so the code hadn't actually been checked in at the time. I'll be committing my changes on the PlaceSearch branch when I've done a bit more testing, and tidied the code up a little bit (there are variables to be renamed, debugging code to be removed etc.) - I'm hoping to push the changes sometime on Sunday evening (UK time).

As I mentioned in issue #22, I messed up when I was creating my branches...
Some of the features in the PlaceSearch branch require a newer version of Qt than Kotelnik's original widget, and I was worried about breaking the widget for users running old distros (e.g. Kubuntu 18.04.LTS) so soon after fixing it.
